sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java

```diff
@@ -97,5 +99,18 @@ public void start() {
             }
           });
     }
+
+    initMetrics(profitabilityConfiguration());
+  }
+
+  private void initMetrics(final LineaProfitabilityConfiguration lineaProfitabilityConfiguration) {
+    final var confLabelledGauge =
+        metricsSystem.createLabelledGauge(
+            LineaMetricCategory.PROFITABILITY,
+            "conf",
+            "Profitability configuration values at runtime",
+            "field");
+    confLabelledGauge.labels(lineaProfitabilityConfiguration::fixedCostWei, "fixed_cost_wei");
+    confLabelledGauge.labels(lineaProfitabilityConfiguration::variableCostWei, "variable_cost_wei");
```

Comment:
Do we need to do the same for extraData content eth gas price?

Reply:
waiting for this https://github.com/Consensys/protocol-misc/issues/953#issuecomment-2431695558 to understand what we mean by _gas price_

Reply:
done, metrics exposed for eth gas price as well